### DISCUSSION TITLE
Does this tool work with quanitzed models?

### DISCUSSION THREAD

**Discussion by bartowski1182** [vllm-project/vllm#482] [Q&A]
Haven't found any reference so I assume no, but wanted to check. would be really cool but also presumably insane levels of work

**Comment by JacobGoldenArt** [comment]
That's exactly what I was looking into. Specifically GPTQ models like: https://huggingface.co/TheBloke/based-30B-GPTQ

**Comment by gesanqiu** [comment]
I also wondering how to start the quantize work. Looking forward to some helps.

**Comment by 2023edu** [comment]
looking forward to update

**Comment by hongqing1986** [comment]
Looking forward to support quantized models.
